### DISCUSSION TITLE
What is the height of the tallest 🌻 ever recorded?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1247] [Galaxy Trivia] [answered]
**Bonus:** Which cooking ingredient comes from a strain of orchids? 🌺

**Comment by peterbaccaro** [comment, marked as answer]
The world record for tallest sunflower is **30 feet, one inch**, set in Westfalen, Germany in 2016.

**Vanilla** is the cooking ingredient that comes from a strain of orchid.
